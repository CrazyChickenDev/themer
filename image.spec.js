const Color = require('color');
const {colorPixel} = require('./image.js');

describe('color pixel renderer', () => {
  it('should render a single-pixel PNG image buffer of the given color', done => {
    colorPixel(Color('teal')).then(data => {
      expect(data.toString('base64')).toMatchSnapshot();
      done();
    });
  });
});
