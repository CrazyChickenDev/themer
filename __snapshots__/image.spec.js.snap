// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`color pixel renderer should render a single-pixel PNG image buffer of the given color 1`] = `"iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAAAklEQVR4AewaftIAAAALSURBVGNgaGj4DwADhAIA8DXq4gAAAABJRU5ErkJggg=="`;
